src/collectors/StopJourneyActionCollector.spec.js

```diff
@@ -0,0 +1,10 @@
+import StopJourneyActionCollector from './StopJourneyActionCollector';
+
+describe("StopJourneyActionCollector", () => {
+  test("test collector sets journeyAction stop on event", async () => {
+    let collector = new StopJourneyActionCollector();
```

Comment:
@ashvince COVERAGE - You are missing the existing properties sent on the object.